lutris/game.py

```diff
@@ -320,6 +322,21 @@ def do_play(self, prelaunched, _error=None):
             show_obnoxious_process_monitor_message()
         process_watch = not monitoring_disabled
 
+        if self.runner.system_config.get('disable_compositor'):
+            if os.environ.get('DESKTOP_SESSION') == "plasma":
```

Comment:
Could the block below be implemented in it's own method?

Reply:
Done.